src/TrueOrFalse.Frontend.Web.Nuxt/components/topic/content/grid/item/item.vue

```diff
@@ -125,7 +125,10 @@ editTopicRelationStore.$onAction(({ after, name }) => {
     if (name == 'addTopic') {
         after((result) => {
             if (result.parentId == props.topic.id) {
-                addGridItem(result.childId)
+                if (children.value.some(c => c.id == result.childId))
+                    reloadGridItem(result.childId)
+                else
+                    addGridItem(result.childId)
             }
         })
     }
```

Comment:
A better name would help. Consider the use of function/library functions. This exact statement is duplicated at least once. Refactor into a method.